### DISCUSSION TITLE
Compilation issue

### DISCUSSION THREAD

**Discussion by MahmoudKnefati** [cpeditor/cpeditor#1170] [Q&A] [answered]
I installed g++ on my windows and I added it to my path but when I start compiling it shows me "No such file or directory" what should I do ? Please help

**Comment by ouuan** [comment]
What's the complete error message?

**Reply by MahmoudKnefati** [reply]
C:/Users/SrajAldeen Company/AppData/Local/Temp/CP Editor-LJIoGA/sol.cpp:1:9: fatal error: iostream.h: No such file or directory
    1 | #include<iostream.h>
      |         ^~~~~~~~~~~~
compilation terminated.

**Reply by ouuan** [reply]
There is no `iostream.h`. Use `#include <iostream>` instead.

**Reply by MahmoudKnefati** [reply]
longer error message had written 
In file included from c:\program files\mingw\include\wchar.h:62,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\cwchar:44,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\bits\postypes.h:40,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\iosfwd:40,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\ios:38,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\ostream:38,
                 from c:\program files\mingw\lib\gcc\mingw32\9.2.0\include\c++\iostream:39,
                 from C:/Users/SrajAldeen Company/AppData/Local/Temp/CP Editor-LJIoGA/sol.cpp:1:
c:\program files\mingw\include\wctype.h:87:10: fatal error: stddef.h: No such file or directory
   87 | #include <stddef.h>
      |          ^~~~~~~~~~
compilation terminated. 
]

**Comment by ouuan** [comment]
Compilation errors are not caused by CP Editor. It could be related to the compiler used by CP Editor on your machine, though. I'd like to help but please try searching online first.

**Comment by ouuan** [comment]
However, you can try using the CP Editor versions with "with-gcc" in their names.

**Reply by MahmoudKnefati** [reply, marked as answer]
it has been solved .. I was adding the compiler to wrong path
